schemas/tax_delinquency_schema.json

```diff
@@ -0,0 +1,308 @@
+     "type": "number",
+     "constraints": {
+       "required": true
+     }
+   },
+   {
+     "name": "Total_Due",
+     "type": "number",
+     "constraints": {
+       "required": true
+     }
+   },
+   {
+     "name": "Num_Yrs_Owed",
+     "type": "integer",
+     "constraints": {
+       "required": true
+     }
+   },
+   {
+     "name": "Most_Recent_Yr_Owed",
+     "type": "year",
+     "constraints": {
+       "required": true
+	    }
+   },
+   {
+     "name": "Oldest_Yr_Owed",
+     "type": "year",
+     "constraints": {
+       "required": true
+     }
+	},
+   {
+     "name": "Most_Recent_Payment_Date",
+     "type": "string"
+   },
+   {
+     "name": "Return_Mail",
+     "type": "boolean"
+   },
+   {
+     "name": "Coll_Agency_Num_Yrs",
+     "type": "integer"
+   },
+   {
+     "name": "Coll_Agency_Most_Recent_Yr",
+     "type": "year",
+     "constraints": {
+       "required": true
+     }
+   },
+   {
+     "name": "Coll_Agency_Oldest_Yr",
+     "type": "year"
+   },
+   {
+     "name": "Coll_Agency_Principal_Owed",
+     "type": "number"
+   },
+   {
+     "name": "Coll_Agency_Total_Owed",
+     "type": "number"
+   },
+   {
+     "name": "Yr_of_Last_Assessment",
+     "type": "year",
+     "constraints": {
+       "required": true
+     }
+   },
+   {
+     "name": "Total_Assessment",
+     "type": "integer"
+   },
+   {
+     "name": "Taxable_Assessment",
+     "type": "integer"
+   },
+   {
+     "name": "Exempt_Abate_Assessment",
+     "type": "integer"
+   },
+   {
+     "name": "Homestead_Value",
+     "type": "number"
+   },
+   {
+     "name": "Net_Tax_Value_After_Hmstd",
+     "type": "number"
+   },
+   {
+     "name": "Building_Code",
+     "type": "string"
+   },
+   {
+     "name": "Detail_Bld_Description",
+     "type": "string"
+   },
+   {
+     "name": "General_Building_Description",
+     "type": "string"
+   },
+   {
+     "name": "Building_Category",
+     "type": "string"
+   },
+   {
+     "name": "Property_Address",
+     "type": "string",
+     "constraints": {
+       "required": true
+     }
+   },
+   {
+     "name": "City",
+     "type": "string",
+     "constraints": {
+       "required": true
+     }
+   },
+   {
+     "name": "State",
+     "type": "string",
+     "constraints": {
+       "required": true
+     }
+   },
+   {
+     "name": "Zip_Code",
+     "type": "integer",
+     "constraints": {
+       "required": true
+     }
+   },
+   {
+     "name": "Co_owner",
+     "type": "string"
+   },
+   {
+     "name": "Mailing_Address",
+     "type": "string"
+   },
+   {
+     "name": "Mailing_City",
+     "type": "string"
+   },
+   {
+     "name": "Mailing_State",
+     "type": "string"
+   },
+   {
+     "name": "Mailing_Zip",
+     "type": "string"
+   },
+   {
+     "name": "Council_District",
+     "type": "integer"
+   },
+   {
+     "name": "X_Long",
+     "type": "number",
+     "constraints": {
+       "required": true
+     }
+   },
+   {
+     "name": "Y_Lat",

```

Comment:
Do we want to drop `Y_Lat` and `X_Long` during transformation or is that done during geocoding? Remember, these are the state projection lat lngs.